crypto/CMakeLists.txt

```diff
@@ -323,9 +323,9 @@ if(ENABLE_DILITHIUM)
   set(
     DILITHIUM_SOURCES
 
-    dilithium/pqdsa.c
-    dilithium/p_pqdsa.c
-    dilithium/p_pqdsa_asn1.c
+    fipsmodule/pqdsa/pqdsa.c
```

Comment:
these need to be added to `bcm.c` that defines the code boundary of the fipsmodule. `DILITHIUM_SOURCES` is not part of the fipsmodule.

Reply:
added in [f4a5a75](https://github.com/aws/aws-lc/pull/2032/commits/f4a5a754d97a23887762db35e07b31f11dc9b4ad)